grass7/gui/wxpython/wx.metadata/m.csw.update/m.csw.update.py

```diff
@@ -0,0 +1,1401 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+############################################################################
+#
+# MODULE:       m.csw.update
+#
+# AUTHOR(S):    Tomas Zigo <tomas.zigo slovanet.sk>
+#
+# PURPOSE:      Update csw connections resources candicates
+#
+# COPYRIGHT:    (C) 2020 by Tomas Zigo, and the GRASS Development Team
+#
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+#%module
+#% description: Update catalogue service for the web connections resources candidates
```

Comment:
```suggestion
#% description: Update catalogue service for the web connections resources candidates.
```

Reply:
In the parser message, I'd write all acronyms with capital letters: XML, URL, XSD, ...

Reply:
It's already fixed.